setup.py

```diff
@@ -13,10 +13,15 @@
       classifiers=[
           'License :: OSI Approved :: MIT License',
           'Programming Language :: Python :: 3',
+          'Programming Language :: Python :: 3.6',
+          'Programming Language :: Python :: 3.7',
+          'Programming Language :: Python :: 3.8',
+          'Programming Language :: Python :: 3.9',
```

Comment:
Too hard to maintain in my opinion. Please remove and I can merge the python_requires addition.